surefire-report-parser/src/main/java/org/apache/maven/plugins/surefire/report/ReportTestCase.java

```diff
@@ -156,6 +158,15 @@ public boolean hasFailure()
         return hasFailure;
     }
 
+    public boolean isError() {
```

Comment:
You are not calling the getter in `SurefireReportGenerator`.

Reply:
> You are not calling the getter in SurefireReportGenerator

I just committed something in that direction but not sure about the modification of the ID in the generated report...